consent-ui/src/app/common/enter-sca/enter-sca.component.html

```diff
@@ -7,21 +7,21 @@ <h2>Input confirmation number</h2>
           We have sent you the confirmation number. Please check your email and fill in the field below.
         </p>
         <div class="form-group">
-          <label class="text-secondary" for="tan">SCA confirmation number:</label><br />
+          <label class="text-secondary" for="sca">SCA confirmation number:</label><br />
```

Comment:
Actually it is TAN confirmation, so it is better to rename SCA -> TAN

Reply:
renamed